x-pack/plugins/actions/server/create_execute_function.ts

```diff
@@ -113,10 +113,15 @@ export function createBulkExecutionEnqueuerFunction({
       inMemoryConnectors,
       connectorIds
     );
+    const skippedConnectors: string[] = [];
 
-    connectors.forEach((c) => {
+    for (const c of connectors) {
       const { id, connector, isInMemory } = c;
-      validateCanActionBeUsed(connector);
+
+      if (connector.isMissingSecrets) {
+        skippedConnectors.push(id);
```

Comment:
It seems like the only thing we used `skippedConnectors` for is for calculating `actionsWithoutSkippedConnectors`.  We might as well do both here, but rename `actionsWithoutSkippedConnectors` to `runnableConnectors` or something.

Also seems like we should log something about the skipped connectors.  Maybe we can just do that instead of collecting them in an array ... (assuming we collect the runnable ones).

Reply:
done